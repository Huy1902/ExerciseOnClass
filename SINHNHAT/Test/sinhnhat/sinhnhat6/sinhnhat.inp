215 130
18718 15313 19624 17911 19727 7215 19564 18087 7305 13005 18743 3087 4230 16954 5414 2319 6628 17117 1656 5464 9313 11452 14424 9841 12621 15451 14871 9435 5245 19828 14044 19474 17104 1399 3910 2882 18882 9978 2002 13659 5010 7047 2845 6318 4653 156 10419 8811 4937 11923 16933 5978 9217 13863 3408 6974 8324 11740 17233 9678 1434 13081 18984 3496 10526 16018 4910 8493 6587 14505 11138 3975 9661 11747 18892 16429 6998 14087 17487 13330 15719 5808 12779 18647 11670 827 3435 1182 4925 9495 13349 6942 5847 13515 12308 3908 13039 1878 5981 12716 2792 18256 17451 12865 16599 3628 14707 5811 3644 13861 6168 11567 9137 10958 12213 2344 3246 4121 15829 19053 3725 16493 2590 13506 1138 12063 222 2904 15626 9890 10863 9693 12682 4394 5138 12797 16480 14208 5192 4405 1681 10918 17856 15463 1950 13939 13295 12300 16436 9599 308 11377 4662 19166 12493 15311 13917 16522 6466 11035 17018 15334 18865 17116 10844 1547 5838 7418 10014 4334 14550 16519 12981 5992 14590 13401 2491 1294 3174 13199 2470 18551 467 18971 18458 5339 2775 16348 4427 15006 3633 18116 17377 18354 13788 11510 13366 9059 4583 2613 10189 14912 4034 3795 14726 15575 1041 4995 3046 16555 13607 2879 16601 2704 17404 