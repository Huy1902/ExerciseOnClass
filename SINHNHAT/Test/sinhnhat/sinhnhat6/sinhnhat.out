1860224
1 2 3 4 5 7 8 10 11 14 18 21 22 23 24 25 26 27 28 30 31 32 33 37 38 40 47 48 50 51 53 54 57 58 59 60 62 63 65 66 68 70 71 73 74 75 76 78 79 80 81 83 84 85 90 91 94 95 97 100 102 103 104 105 107 110 112 113 114 115 119 120 122 124 126 129 130 131 132 133 136 137 138 142 143 144 146 147 148 149 150 152 154 155 156 157 158 160 161 162 163 164 165 169 171 172 173 175 176 180 182 184 185 188 190 192 193 194 195 196 197 198 201 202 205 206 210 211 213 215 